### ISSUE TITLE
Is this thing on?

### ISSUE THREAD

**Issue by rberger** [stormpath/stormpath-sdk-go#2]
Readme says Status: IN DEVELOPMENT!

But that was from a while a go. Wondering if this is the one or if there is another Go sdk to use with Stormpath.

Thanks!

**Comment by lhazlewood** [comment]
It is very much experimental.  A community-contributed Golang SDK here is much more robust at the moment:

https://github.com/jarias/stormpath-sdk-go

Go is not currently officially supported by Stormpath, but we hope to eventually.  We do not have a timeline as to when this would occur.  HTH!

**Comment by rdegges** [comment]
For what it's worth, I've been using this for some projects and it's rock solid =)

**Comment by lhazlewood** [comment]
It may be, but it is not supported by Stormpath (i.e. we will not be able to support this on support.stormpath.com).  Please use at your own risk until we have an official release.

**Comment by rberger** [comment]
Understood about support. At least its something we can start with. Ability to use it with Go is critical for us.

Hopefully it will become officially supported soon though. Go is becoming the “go too” language for ops :-)

Rob

**Comment by lhazlewood** [comment]
Indeed!  Let us know if you try the jarias version as well - I'm happy to report he's working closely with the Stormpath core engineering team and has made a lot of progress on that as well.  I think it is much more comprehensive than the one in Stormpath's GitHub repo.  @jarias any update on yours?

**Comment by jarias** [comment]
@rberger you can check https://github.com/jarias/stormpath-sdk-go it has almost a 100% of Stormpath's API implemented, things that are missing are still in beta in Stormpath so it should be fully usable.

If you need some help just let me know.
